docs/source/API/overview_development.rst

```diff
@@ -0,0 +1,56 @@
+Overview
+========
+
+The development section is meant for the more advanced user of ASReview. It
```

Comment:
you might want to briefly describe all subsections and not only one. So something like: In this section an overview of the structure of ASReview is explained....

Reply:
I am not sure what is meant here :)